base/docker/build.sh

```diff
@@ -8,6 +8,6 @@ IMG_NAME="aperturedata/workflows-base"
 
 # Build docker image
 cd $DIR
-docker build --no-cache -t ${IMG_NAME} .
+docker build -t ${IMG_NAME} .
```

Comment:
I think you want to revert this before merging

Reply:
will do.